<launch>
  <include file="$(find realsense2_camera)/launch/rs_rgbd.launch">
    <arg name="fisheye_fps" value="15"/>
    <arg name="depth_fps" value="15"/>
    <arg name="infra_fps" value="15"/>
    <arg name="color_fps" value="15"/>
  </include>
  <node pkg="pointcloud_smasher" type="pointcloud_smasher" name="pointcloud_smasher">
    <param name = "CAMERA_ANGLE" value = "0"/>
    <param name = "CAMERA_HEIGHT" value = "0.27"/>
    <param name = "CAMERA_X_LIMIT" value = "5.0"/>
    <param name = "ANGLE_THRESHOLD" value = "0.785"/>
  </node>
</launch>
